### DISCUSSION TITLE
My Git repository and data is been removed & not showing in github

### DISCUSSION THREAD

**Discussion by SouvikAsansolution** [community/community#23131] [Projects and Issues]
<p>Hi, I am Souvik Bhattacharji. My GitHub account is SouvikAsansolution and I have a serious problem that is my GitHub data have been stolen from my account and not showing in the reprosatory. It has some important source code  &amp; analysis that is placed in private mode. but I found it strange that the data is missing and nighter has any track of it. I don’t know what to do next. and I was surprised to see that a reputed company like git has this kind of problem. please help and advice about this matter by the support team.<br>
and do the needful so that this kind of problem farther not happen.<br>
Regards<br>
Souvik</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/souvikasansolution">@SouvikAsansolution</a> and welcome!</p>
<p>I recommend contacting our <a href="https://support.github.com/contact">Support team</a> as they are the most well-equipped to assist in account-specific situations like this.</p>
<p>Best of luck!</p>
